### PR TITLE
Allow publishing only CLEAN or only INFECTED results

### PR DESCRIPTION
**Pull request by chrisgilmerproj**
This change allows only publishing to SNS if the scan result is CLEAN or INFECTED. By default it will publish both which was the previous default. 

Background: My company doesn't want to know if the results are CLEAN. They only want notifications if the results are INFECTED which will kick off additional work for our infrastructure team.

The alternative approach I considered was having an SNS topic ARN for CLEAN and one for INFECTED so that they could be treated differently. However this seemed like it would be backwards incompatible and I don't know the project's tolerance for that kind of change.

### CODE REVIEW

File: scan.py [hunk 1]
```diff
@@ -119,9 +119,17 @@ def sns_start_scan(s3_object):
         MessageStructure="json"
     )
 
+
 def sns_scan_results(s3_object, result):
+    # Don't publish if SNS ARN has been supplied
```
Comment (jaygorrell) [inline scan.py]:
```suggestion
    # Don't publish if SNS ARN has not been supplied
```

Comment (chrisgilmerproj) [reply scan.py]: resolved in 5a4929e

**Comment by chrisgilmerproj** [comment]
I think this closes #29

**Comment by chrisgilmerproj** [comment]
This may also be related to #65 .

**Pull request merged**